network.py

```diff
@@ -79,18 +82,28 @@ def run(self):
     def _read_packet(self, data):
         parsed = json.loads(data.decode())
         if parsed["sender"] == "vision":
-            self.angle = parsed["angle"]
+            key = parsed["object"]
```

Comment:
Noting here, we need to catch more exceptions in the `try/except` above, for instance, we need to catch `JSONDecodeError`, which means maybe we should just have a catch-all `Exception` case above so the network case doesn't fail.
Alternately, we should catch errors inside `_read_packet` and re-throw a uniform error.